server.js

```diff
@@ -142,24 +142,18 @@ app.get('/intranet', function(req, res) {
   }
 
   request({
-    url: `${SERVICES_URL}/credits/users/${req.session.student.netid}`,
+    url: `${SERVICES_URL}/merch/users/${req.session.student.netid}`,
```

Comment:
Wait, so merch provides the user's balance? I guess that's fine, just seems a bit strange to wrap one service in another.